guides/doc-Administering_Project/master.adoc

```diff
@@ -11,6 +11,8 @@ endif::[]
 ifndef::HideDocumentOnStable[]
 
 ifdef::satellite[]
+include::common/modules/con_making-open-source-more-inclusive.adoc[]
```

Comment:
I also wonder if this needs a level offset like line 16. Your call.

Reply:
Yes, it works better with a level offset. We used to use no level offset and Asciidoctor generated a preface. Now, it generates a "Part 1", which is not the desired outcome. Will fix.